gcloud/storage/batch.py

```diff
@@ -170,10 +170,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 def _unpack_batch_response(response, content):
     """Convert response, content -> [(status, reason, payload)]."""
     parser = Parser()
-    faux_message = ('Content-Type: %s\nMIME-Version: 1.0\n\n%s' %
-                    (response['content-type'], content))
-
-    message = parser.parsestr(faux_message)
+    if not isinstance(content, six.binary_type):
```

Comment:
Coercing to bytes seems to be awkward:  we want `insinstance(faux_message, str)` to be true, right?

Couldn't we write a helper function (e.g., `to_native_str()`), which would Do the Right Thing(TM) here without this dance?